README.rst

```diff
@@ -1,3 +1,16 @@
+Report link  
+
+https://docs.google.com/document/d/1qTW-mjOnhJ3-R_6riB_1sUDna1cKf4MjGjzGgeK9zcE/edit?usp=sharing
+
+Pull requests
```

Comment:
Unfortunately, we can't merge this with this in there

Reply:
Whoops! I forgot that the changes would show up here. I'll revert it ASAP